README.md

```diff
@@ -973,9 +982,10 @@ Clients for commercial social platforms that had their API access cut off in a w
 
 - [An Anime Game launcher](https://github.com/an-anime-team/an-anime-game-launcher) - Genshin Impact launcher for Linux with telemetry disabling `#rust` `#gtk4` `#relm4` `#libadwaita`.
 - [Cartridges](https://github.com/kra-mo/cartridges) - Game launcher with Steam, Lutris, Heroic, Legendary, Bottles, itch and RetroArch library import `#python` `#gtk4` `#libadwaita`.
-- [Lutris](https://lutris.net) - Game launcher covering most gaming systems `#python` `#gtk3`.
 - [Gameeky](https://github.com/tchx84/gameeky) - Application to create and play games without any code for young learners and educators `#python` `#gtk4` `#libadwaita`.
+- [Highscore](https://gitlab.gnome.org/World/highscore) - Retro gaming application for the GNOME desktop, formerly gnome-games `#vala` `#gtk4` `#libadwaita`.
```

Comment:
Is this mature enough to be added on the list?

Reply:
You are right, highscore seems to be not release-ready yet. It is probably better to leave it out save it for later when it will be ready.
The developer has been very active lately, but highscore is not yet on flathub, neither there is a release yet. 
I found this reference (May 2023): "In short: Highscore is still in development, and currently not very active. It will be on flathub once it’s done." https://discussion.fedoraproject.org/t/what-happened-to-gnome-games-in-flathub/82747/6